src/main/java/com/kodilla/ecommercee/dto/Product.java

```diff
@@ -4,13 +4,80 @@
 import lombok.Getter;
 import lombok.NoArgsConstructor;
 
-@AllArgsConstructor
-@NoArgsConstructor
-@Getter
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.Id;
+import javax.persistence.Table;
+import javax.validation.constraints.NotNull;
+
+//@AllArgsConstructor
```

Comment:
Moim zdaniem mozemy spokojnie tutaj uzyc lmobok, tak samo zrobic w moim comicie dla User. Jedyne co by bylo dodac to konstruktor ze wszystkimi polami procz id.